### ISSUE TITLE
need ability to disable web-page auto refresh

### ISSUE THREAD

**Issue by Master-Frodo** [mozilla-mobile/fenix#20459]
need to be able to read long documents or finish videos before web page enabled auto refresh kicks in.  The accessibility.blockautorefresh option not accessible in android firefox.

**Comment by kbrosnan** [comment]
If you need access to that setting you can use https://play.google.com/store/apps/details?id=org.mozilla.firefox_beta or https://play.google.com/store/apps/details?id=org.mozilla.fenix

**Comment by Master-Frodo** [comment]
Neither of those options allowed any change to autorefresh.  Your solution does not work.  They both allowed me to open about:config, but neither had the option to change blockautorefresh.  How does this get re-opened?
